app/presenters/track.rb

```diff
@@ -0,0 +1,39 @@
+module ExercismWeb
+  module Presenters
+    class Track
+
+      def initialize(track_id)
+        @track_id = track_id
+        @track = Trackler.tracks[track_id]
```

Comment:
If we define this in a `def track` method, we could stub that in a test to be a tiny object that is missing the documentation.